src/Elcodi/Bundle/CartBundle/Resources/config/doctrine/Order.orm.yml

```diff
@@ -49,6 +49,13 @@ Elcodi\Component\Cart\Entity\Order:
         updatedAt:
             column: updated_at
             type: datetime
+        taxAmount:
```

Comment:
You can just push a new commit to your branch, it will get updated here automatically (even with a `--force`)